minikube-gradle-plugin/src/test/java/com/google/cloud/tools/minikube/MinikubeExtensionTest.java

```diff
@@ -17,48 +17,103 @@
 package com.google.cloud.tools.minikube;
 
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 import com.google.cloud.tools.minikube.util.CommandExecutor;
 import com.google.cloud.tools.minikube.util.CommandExecutorFactory;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Ah, this part needs to be expanded back out too.

Reply:
Done.
Thanks for the thorough review. Was quite a good induction to Google's code style practices; I was not aware of the tools.